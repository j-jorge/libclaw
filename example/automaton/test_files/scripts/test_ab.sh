#!/bin/sh

./ex-automaton "test_files/automaton_file/ab.txt" ababababababa ababab bababa bababab a b ""
