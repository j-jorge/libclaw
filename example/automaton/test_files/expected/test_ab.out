A = { a, b }
I = { 1 }
F = { 3 }
E = { 1, 2, 3 }
T = {
1 -> { (2, a) }
2 -> { (3, b) }
3 -> { (2, a) }
}

ababababababa: not valid
ababab: valid
bababa: not valid
bababab: not valid
a: not valid
b: not valid
: not valid
